### DISCUSSION TITLE
bun not running local package bin

### DISCUSSION THREAD

**Discussion by jdion84** [oven-sh/bun#5232] [Q&A]
bun not running local package bin, heres what i did...

create file `my_modules/bunclitest/package.json`:

```js
{
  "name": "bunclitest",
  "module": "index.ts",
  "type": "module",
  "bin": {
    "bunclitest": "bin.js"
  }
}
```

create file `my_modules/bunclitest/bin.js`:

```js
#!/usr/bin/env node

console.log("hello bin");
```

open `my_modules` dir in terminal and run `bun link`

open project root folder and run `bun link bunclitest --save`

run `bunx myclitest`

nothing is outputted to console.

**Comment by melroy89** [comment]
Of the record: `#!/usr/bin/env node` will run the script in NodeJS when using `bun run ...` or via CLI command line. The shebang should ideally be `bun`.
